testsuite/MDAnalysisTests/analysis/test_hbonds.py

```diff
@@ -59,6 +59,16 @@ def setUp(self):
             'acceptor_resnm': np.array(['ALA', 'ALA', 'ALA', 'ALA', 'ALA', 'PRO', 'ALA', 'ALA'], dtype='U4'),
             }
 
+    #issue 1229        
+    def test_get_bonded_hydrogen_list(self, **kwargs):
+    	kw = self.kwargs.copy()
+    	kw.update(kwargs)
+    	h = MDAnalysis.analysis.hbonds.HydrogenBondAnalysis(self.universe, **kw)
+    	self.universe.atoms[0].index = -2
```

Comment:
Initially I tried doing the same thing by picking the last atom of the universe i.e self.universe.atoms[-1] this raises an exception but self.universe.atoms[1] also raises the exception so i shifted to that . so should i simply pick the last atom of universe ? (